docs/integrations/sources/instagram.md

```diff
@@ -146,7 +146,8 @@ for more information.
 
 | Version    | Date       | Pull Request                                             | Subject                                                                                                                                                                |
 |:-----------|:-----------|:---------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 4.1.0-rc.3 | 2025-07-10 | [62902](https://github.com/airbytehq/airbyte/pull/62902)     | Revert add `views` metric to `StoryInsights` and `MediaInsights` streams.                                                                                              |
+| 4.2.0-rc.1 | 2025-07-15 | [tbd](https://github.com/airbytehq/airbyte/pull/tbd)     | Migrate to manifest-only format.                                                                                                                                       |
```

Comment:
plan will be to finalize the current rc3 release monday and this will be the subsequent minor version bump

Reply:
yep, was waiting for the previous release to finalize cuz i'm tired of merge conflicts in life. but will have that in the final PR